torch/csrc/jit/stack_functions.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
```

Comment:
nit: can we just call this file `stack.h`? it doesn’t only have helpers for it, but the definition too.